watch/inotify.go

```diff
@@ -96,12 +96,14 @@ func (fw *InotifyFileWatcher) ChangeEvents(t *tomb.Tomb, pos int64) (*FileChange
 			}
 
 			switch {
-			//With an open fd, unlink(fd) - inotify returns IN_ATTRIB (==fsnotify.Chmod)
 			case evt.Op&fsnotify.Chmod == fsnotify.Chmod:
-				if _, err := os.Stat(fw.Filename); err != nil {
-					if !os.IsNotExist(err) {
-						return
-					}
+				_, err = os.Stat(fw.Filename)
+				if err == nil {
+					continue
+				}
+				// With an open fd, unlink(fd) - inotify returns IN_ATTRIB (==fsnotify.Chmod)
+				if !os.IsNotExist(err) {
```

Comment:
Wouldn't we be swallowing the error, if the error is something other than `NotExist`?